packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx

```diff
@@ -89,16 +91,22 @@ export class PersonaCoin extends React.Component<IPersonaProps, IPersonaState> {
 
     let size = this.props.size as PersonaSize;
     let divProps = getNativeProps(this.props, divProperties);
+    let coinSizeStyle = coinSize ? { width: coinSize, height: coinSize } : undefined;
 
     initialsColor = initialsColor !== undefined && initialsColor !== null ? initialsColor : this._getColorFromName(primaryText);
 
     return (
       <div
         { ...divProps }
         className={ css('ms-Persona', styles.root, className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence as PersonaPresenceEnum], this.props.showSecondaryText ? styles.showSecondaryText : '') }
+        style={ coinSize ? { height: coinSize, minWidth: coinSize } : undefined }
       >
         { size !== PersonaSize.tiny && (
-          <div className={ css('ms-Persona-imageArea', styles.imageArea) }>
+          <div
+            { ...coinProps }
+            className={ css('ms-Persona-imageArea', styles.imageArea) }
+            style={ coinSizeStyle }
```

Comment:
mergeStyles

Reply:
Using merge style cannot cover this condition very well. The style of width and height will be override by other class name which also defined size.